Client/qtTeamTalk/mainwindow.cpp

```diff
@@ -1100,6 +1131,13 @@ void MainWindow::processTTMessage(const TTMessage& msg)
                m_talking.size() == 0)
                 m_sysicon->setIcon(QIcon(APPTRAYICON_CON));
         }
+
+        if(user.uUserState & USERSTATE_MEDIAFILE_AUDIO && user.nUserID != TT_GetMyUserID(ttInst))
+        {
+            User nameuser;

```

Comment:
You don't need to extract user as "nameuser". Just use existing "user".

Btw, if you have a C-struct in C++ then initialize it like this: "User user = {};" Then all data in the struct is initialized to zero (this is important for strings since they are zero terminated).